app/controllers/crops_controller.rb

```diff
@@ -61,14 +61,20 @@ def hierarchy
   # GET /crops/search
   def search
     @all_matches = Crop.search(params[:search])
-    exact_match = Crop.find_by_name(params[:search])
+    exact_match = Crop.find_by(name: params[:search])
+    exact_search = params[:exact_search] || false
+
     if exact_match
       @all_matches.delete(exact_match)
```

Comment:
This seems to delete from the actual database in development. I think this is because we're not using Elasticsearch in the dev env and so `search` is returning an `ActiveRecord::Relation`, not a regular array. It looks like calling search results in queries like this for example:

`DELETE FROM "crops" WHERE (name ILIKE '%corn%') AND "crops"."id" = 61`

Not sure how to debug this.